src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -385,7 +385,7 @@ internal static DiagnosticInfo ToFileReadDiagnostics(CommonMessageProvider messa
             return diagnosticInfo;
         }
 
-        public bool ReportErrors(IEnumerable<Diagnostic> diagnostics, TextWriter consoleOutput, ErrorLogger errorLoggerOpt)
+        internal bool ReportErrors(IEnumerable<Diagnostic> diagnostics, TextWriter consoleOutput, ErrorLogger errorLoggerOpt)
```

Comment:
Could IEnumerable be switched to ImmutableArray, or are there callers that still need IEnumerable?

Reply:
They still use `IEnumerable` unfortunately.